data/files.sql

```diff
@@ -31,10 +31,10 @@ CREATE TABLE access_method (
         PRIMARY KEY (id), 
         FOREIGN KEY(drs_object_id) REFERENCES drs_object (id)
 );
-INSERT INTO access_method VALUES(1,'NA18537.vcf.gz.tbi','s3','docker.localhost:9000/samples/NA18537.vcf.gz.tbi','','','[]');
-INSERT INTO access_method VALUES(2,'NA18537.vcf.gz','s3','docker.localhost:9000/samples/NA18537.vcf.gz','','','[]');
-INSERT INTO access_method VALUES(3,'NA20787.vcf.gz.tbi','s3','docker.localhost:9000/samples/NA20787.vcf.gz.tbi','','','[]');
-INSERT INTO access_method VALUES(4,'NA20787.vcf.gz','s3','docker.localhost:9000/samples/NA20787.vcf.gz','','','[]');
+INSERT INTO access_method VALUES(1,'NA18537.vcf.gz.tbi','s3','s3.amazonaws.com/daisietestbucket1/test/NA18537.vcf.gz.tbi','','','[]');
```

Comment:
Should not hardcode private bucket

Reply:
It's not meant to be a private bucket; I just haven't figured out how to set up the permissions correctly for a separate user. I intend for us to have a credential that can access these files for testing.